packages/gatsby-cli/src/reporter/reporter.ts

```diff
@@ -158,6 +158,7 @@ class Reporter {
 
     if (structuredError) {
       reporterActions.createLog(structuredError)
+      trackError(`GENERAL_ERROR`, { error: structuredError })
```

Comment:
general or user?

Reply:
I think this is "any" error, maybe generic is better term